tests/test/sgx/test_sgx_enclave.cpp

```diff
@@ -0,0 +1,49 @@
+#include <catch2/catch.hpp>
+
+#include <sgx/WAMREnclave.h>
+#include <sgx/system.h>
+
+namespace tests {
+class SgxTestFixture
+{
+  public:
+    SgxTestFixture() {}
+
+    ~SgxTestFixture() {}
+
+  protected:
+    std::shared_ptr<sgx::WAMREnclave> wamrEnclave = nullptr;
+};
```

Comment:
Is this fixture used anywhere? Might be useful to add the appropriate cleanup shared by all SGX tests and put it in the shared fixtures header file.